hazelcast/src/main/java/com/hazelcast/monitor/TimedMemberState.java

```diff
@@ -190,12 +190,9 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        StringBuilder sb = new StringBuilder("TimedMemberState{\n");
-        sb.append("\t");
+        StringBuilder sb = new StringBuilder("TimedMemberState{\n\t");
```

Comment:
Do we have a newline constant/variable/getter somewhere in Hazelcast? What do you prefer for this? I don't like to put a lot of `System.getProperty("line.separator")` calls into the code.